### PR TITLE
Sample example

### PR DESCRIPTION
**Pull request by lin121291**
- 新增src底下目錄
- 簡單的example

### CODE REVIEW

**Comment by asas1asas200** [comment]
Related issues: #3

**Comment by lin121291** [comment]
ok 謝啦 我晚點push更改

File: .gitignore [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Latex
 
+#npm package
+package-lock.json
+.keep
```
[the remark this answers is not in the document]
Comment (asas1asas200) [reply .gitignore:5]: Use `.keep` instead `tmp.js`

**Comment by lin121291** [comment]
我更新了

**Comment by asas1asas200** [comment]
You should add them back and commit again, otherwise this program cannot build.

And use `.keep` instead `tmp.js`.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,17 @@
...
+
+---
+
+```bash
+# go to code
+$ cd src
+
+# Get npm package
+$ npm build
+
```
Comment (asas1asas200) [inline README.md:13]: This command failed as below:
```
$ node build
node:internal/modules/cjs/loader:998
  throw err;
  ^

Error: Cannot find module '/home/zeng/Desktop/NTOUClassroomBorrowing/src/build'
    at Module._resolveFilename (node:internal/modules/cjs/loader:995:15)
    at Module._load (node:internal/modules/cjs/loader:841:27)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12)
    at node:internal/main/run_main_module:23:47 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}

Node.js v18.11.0
```

Use `npm install` instead.

File: README.md [hunk 3]
```diff
@@ -1,2 +1,17 @@
...
 
+## Quick start
+
+---
+
+```bash
+# go to code
+$ cd src
+
+# Get npm package
```
Comment (asas1asas200) [inline README.md:11]: The README is an important document, so you should write more rigorous, such as consistent capital in each line.

**Review by asas1asas200 — CHANGES_REQUESTED** [review]

**Review by asas1asas200 — APPROVED** [review]

**Pull request merged**